src/state/ThemeReducer.js

```diff
@@ -0,0 +1,10 @@
+const ThemeReducer = (stateTheme, action) => {
+  switch (action.type) {
+    case 'SET_THEME':
+      return { ...stateTheme, theme: action.payload };
+    default:
+      return stateTheme;
+  }
+};
```

Comment:
Same here about naming your action content:
```suggestion
const ThemeReducer = (stateTheme, action) => {
  const { type, theme } = action;
  switch (type) {
    case 'SET_THEME':
      return { ...stateTheme, theme };
    default:
      return stateTheme;
  }
};
```